src/Translation/locales/zh.json

```diff
@@ -3,40 +3,40 @@
     "completed": "已完成",
     "no_activity": "您还没有活动。",
     "pending": "待定",
-    "start": "您可以先在您的设置页面{settings_link}上批准一些MANA"
+    "start": "您可以先在您的设置页面{settings_link}上购买一些 MANA"
   },
   "address": {
-    "its_you": "是你"
+    "its_you": "你的"
   },
   "atlas": {
     "loading": "载入中",
-    "no_owner": "没有所有者",
+    "no_owner": "无主地",
     "on_sale": "出售中：",
     "owner": "所有者：{owner}",
     "road": "路",
     "your_parcel": "您的地块"
   },
   "buy_mana": {
-    "action": "购买更多",
-    "amount_too_big": "Bancor Network的这个数额太大了",
+    "action": "购买",
+    "amount_too_big": "对于 Bancor Network 来说数量过高",
     "available_eth": "可用：{eth} ETH",
-    "not_enough": "你没有足够的ETH来支付这个费用",
-    "pay_with": "付款",
-    "powered_by": "由{bancor_link}提供",
-    "rate": "率",
-    "receive": "收到",
-    "slippage": "滑移",
-    "title": "将ETH转换为MANA"
+    "not_enough": "你没有足够用于支付的 ETH",
+    "pay_with": "需支付",
+    "powered_by": "由 {bancor_link} 提供",
+    "rate": "兑换率",
+    "receive": "你将收到",
+    "slippage": "误差",
+    "title": "使用 ETH 兑换 MANA"
   },
   "color_key": {
     "contribution": "小区份额",
```

Comment:
@leelinliew @oscnet This is referring to parcels of LAND that the user has "contributed" or donated/given to a community district. The community districts are all made up of LAND parcels that various users have grouped together. When you look at the Atlas, any parcels that you have "contributed" to a district are highlighted in a different color. Let me know if this helps to explain it.

Reply:
I see, but  I can't find any such contributed parcels at the atlas even if  I have some district parcels. it's all colored as "district" , Is this normal?